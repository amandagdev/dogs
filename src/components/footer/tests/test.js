import React from 'react'
import { render, screen } from '@testing-library/react'
import Footer from '..'

describe('<Footer/>', () => {
  it('should render the footer', () => {
    const { container } = render(<Footer />)
    const footer = screen.getByText(/footer/i)
    expect(footer).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
