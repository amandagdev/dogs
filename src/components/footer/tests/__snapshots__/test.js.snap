// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['<Footer/> should render the footer 1'] = `
<div>
  footer
</div>
`
